src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj

```diff
@@ -6,6 +6,9 @@
     <PackageTags>aspire integration hosting aws</PackageTags>
     <Description>Add support for provisioning AWS application resources and configuring the AWS SDK for .NET.</Description>
     <NoWarn>$(NoWarn);CS8002</NoWarn> <!-- AWS CDK packages are not signed -->
+    <SignAssembly>true</SignAssembly>
```

Comment:
Should we move the `SignAssembly` and `AssemblyOriginatorKeyFile` to `Directory.Build.props` so we don't forget to set them in the future?

Reply:
Done